core/designsystem/src/main/java/com/hilingual/core/designsystem/theme/Color.kt

```diff
@@ -0,0 +1,11 @@
+package com.hilingual.ui.theme
```

Comment:
P2: theme 이쪽 파일들 
```suggestion
package com.hilingual.core.designsystem.theme
```
이렇게 바꾸라고 뜨는듯!

Reply:
아 맞네요 수정하겟굽